standard/classes.md

```diff
@@ -3174,7 +3174,7 @@ In a *ref_property_body* an expression body consisting of `=>` followed by `ref`
 
 > *Note*: Even though the syntax for accessing a property is the same as that for a field, a property is not classified as a variable. Thus, it is not possible to pass a property as an `in`, `out`, or `ref` argument unless the property is ref-valued and therefore returns a variable reference ([§9.7](variables.md#97-reference-variables-and-returns)). *end note*
 
-When a property declaration includes an `extern` modifier, the property is said to be an ***external property***. Because an external property declaration provides no actual implementation, each of its *accessor_declarations* consists of a semicolon.
+When a property declaration includes an `extern` modifier, the property is said to be an ***external property***. Because an external property declaration provides no actual implementation, each of the *accessor_body*s in its *accessor_declarations* shall be a semicolon.
```

Comment:
This is an odd plural form "*accessor_body*s", but I'll defer to you.

Reply:
Yes it is! However, the grammar rule name is *accessor_body*, singular, and I need the plural, but can't say *accessor_bodies*, as that is not a rule name. In hindsight, I could have said, "each *accessor_body* in its ..."